### PR TITLE
OpenSSL SHA hashing implementation.

### PR DESCRIPTION
**Pull request by pkthapa**

### CODE REVIEW

**Review by arc-vmware — APPROVED** [review]
LGTM!!!

File: util/CMakeLists.txt [hunk 1]
```diff
@@ -76,6 +76,7 @@ set(util_header_files
     include/Digest.hpp
     include/DigestType.hpp
     include/DigestImpl.ipp)
+
```
Comment (arc-vmware) [inline util/CMakeLists.txt:79]: Is this related to this change.

**Review by arc-vmware — CHANGES_REQUESTED** [review]
Keeping a request for change.

**Comment by pkthapa** [comment]
Closing this PR. New PR created here: https://github.com/arc-vmware/concord-bft/pull/2

**Pull request closed without merging**